### PR TITLE
add hideForPhonesOnly to Drawer and revamp props

### PR DESCRIPTION
**Pull request by hobbitronics**
### Added
- hideForPhonesOnly, hideForTablet, miniMenu to Drawer

### Fixed
- bug preventing standard navigation Drawer

### Removed
- hideForMobile from Drawer

### Changed
- all boolean props in Drawer are now false by default

### CODE REVIEW

File: stories/Drawer.stories.svelte [hunk 1]
```diff
@@ -12,25 +12,23 @@
 			{
 				url: './',
         icon: 'account_circle',
-				label: 'see our components in storybook'
+				label: 'reload'
```
Comment (forevermatt) [inline stories/Drawer.stories.svelte:15]: Looks like this file has mixed tabs and spaces, making it slightly harder to read.

**Review by forevermatt — APPROVED** [review]
Looks good

File: stories/Drawer.stories.svelte (hunk 1 above)
Comment (hobbitronics) [reply stories/Drawer.stories.svelte]: reformatting cleared up the weirdness. I wonder if there is a prettier config we could all agree on for js.

Comment (forevermatt) [reply stories/Drawer.stories.svelte]: Sounds worth trying 👍

**Review by forevermatt — APPROVED** [review]

**Pull request merged**